src/ast/mod.rs

```diff
@@ -696,6 +698,7 @@ impl fmt::Display for Expr {
             Expr::Collate { expr, collation } => write!(f, "{} COLLATE {}", expr, collation),
             Expr::Nested(ast) => write!(f, "({})", ast),
             Expr::Value(v) => write!(f, "{}", v),
+            Expr::IntroducedString { introducer, value } => write!(f, "{} {}", introducer, value),
```

Comment:
@mskrzypkows is this supposed to always be spaced?

In simple strings, it seems that there's no space:

```sql
SELECT _latin1'abc';
```

While in strings like a binary string does have spaces:

```sql
SELECT _latin1 X'4D7953514C';
```

Am I missing something?

**OBS:** both examples are from your reference: https://dev.mysql.com/doc/refman/8.0/en/charset-introducer.html

Reply:
The space doesn't matter, it may be both with or without.

Reply:
Testing both versions here: https://github.com/sqlparser-rs/sqlparser-rs/pull/788/files#diff-c8a9c2f5aa6b5c0f71fde332e59342930b2d67697d6fa11b538a7fadbbe1fe78R1302

Reply:
Oh, I see, sorry the confusion.